modules/bg96/cellular_bg96.c

```diff
@@ -356,3 +447,16 @@ CellularError_t Cellular_ModuleEnableUrc( CellularContext_t * pContext )
 }
 
 /*-----------------------------------------------------------*/
+
+void _Cellular_NotifyUdpSocketOpenResult( CellularUrcEvent_t urcEvent,
```

Comment:
This function is not public function and the caller of this function already assume the parameters are checked. This function is only used in cellular_bg96_urc_handler.c.
Suggest to move this function to urc handler file ( cellular_bg96_urc_handler.c )

Reply:
Good suggestion. Moved.